### ISSUE TITLE
Evaluate if we want to inlude openshift2nulecle in ADB

### ISSUE THREAD

**Issue by LalatenduMohanty** [projectatomic/adb-atomic-developer-bundle#235]
Should we include openshift2nulecle[2] in ADB as a RPM or we should expect it as a container?

[1] https://copr.fedorainfracloud.org/coprs/tkral/openshift2nulecle/
[2] https://github.com/kadel/Openshift2Nulecule

**Comment by kadel** [comment]
openshift2nulecule repo got moved to projectatomic org: https://github.com/projectatomic/openshift2nulecule

I also updated it so ti can be easier run it from container using `atomic` cli (see [README.md](https://github.com/projectatomic/openshift2nulecule#running-as-docker-container))

At last cabal call we talked about including this as RPM. 
Now there is also option of running this in container if we change our mind.
If there is no real reason of doing otherwise, I still think that installing it directly to system as RPM is better approach.
